MekHQ/src/mekhq/gui/model/PatientTableModel.java

```diff
@@ -69,7 +69,7 @@ public Component getListCellRendererComponent(
                 boolean isSelected,
                 boolean cellHasFocus) {
             Person p = getElementAt(index);
-            setPortrait(p);
+            setImage(p.getPortrait().getImage(54));
```

Comment:
It's a graphical size scaling value, and is based on the individual graphical setup. That 54 tends to be the best working value in MekHQ is likely because of earlier code reuse, and is not because of anything else (portraits default to 72).

As it's an internal graphical value, like creating a dimension, insets, or anything else really graphical, I don't think it makes any sense to turn into a constant.